src/components/VmDetails/cards/SnapshotsCard/index.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import BaseCard from '../../BaseCard'
+import style from './style.css'
+
+import NewSnapshotModal from './NewSnapshotModal'
+import SnapshotItem from './SnapshotItem'
+
+const Snapshots = ({ snapshots, vmId }) => {
+  const isVmInPreview = !!snapshots.find(snapshot => snapshot.get('status') === 'in_preview')
+  return (
+    <React.Fragment>
+      {
+        !isVmInPreview &&
+        <div className={style['snapshot-create']}>
+          <NewSnapshotModal vmId={vmId} />
+        </div>
+      }
+      {snapshots.sort((a, b) => b.get('date') - a.get('date')).map((snapshot) => (
+        <SnapshotItem key={snapshot.get('id')} snapshot={snapshot} vmId={vmId} isEditing={!isVmInPreview} />
+      ))}
```

Comment:
Can you double check that the new snapshot left icon is the same size as the left status/info icons on the snapshots?  I remember them off by just a bit.